### PR TITLE
[FLINK-26638][connectors/elasticsearch] update the Chinese document accordingly

### PR DESCRIPTION
**Pull request by gaoyunhaii**
## What is the purpose of the change

This PR updates the Chinese document of Elasticsearch connector to keep consistent with the Engish version.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): **no**
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: **no**
  - The serializers: **no**
  - The runtime per-record code paths (performance sensitive): **no**
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: **no**
  - The S3 file system connector: **no**

## Documentation

  - Does this pull request introduce a new feature? **no**
  - If yes, how is the feature documented? **not applicable**

### CODE REVIEW

**Comment by gaoyunhaii** [comment]
Hi @KarmaGYZ Could you have a look at this Chinese document change?

**Review by KarmaGYZ — APPROVED** [review]

**Review by hackergin — COMMENTED** [review]
It looks like there should be spaces here

File: docs/content.zh/docs/connectors/table/elasticsearch.md [hunk 1]
```diff
@@ -130,11 +130,28 @@ CREATE TABLE myUserTable (
...
+        <li>自定义类名称：使用 ActionRequestFailureHandler 的子类进行失败处理。</li>
+      </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><h5>sink.flush-on-checkpoint</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">true</td>
+      <td>Boolean</td>
+      <td>在进行 checkpoint 时是否保证刷出缓冲区中的数据。如果关闭这一选项，在进行checkpoint时 sink 将不再为所有进行
```
[the remark this answers is not in the document]
Comment (gaoyunhaii) [reply docs/content.zh/docs/connectors/table/elasticsearch.md:152]: Hi @hackergin the space is required to separate between Chinese and English words (https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications)

**Comment by gaoyunhaii** [comment]
Thanks @KarmaGYZ @hackergin for the review! Will merge~

**Pull request closed without merging**